airflow/www/views.py

```diff
@@ -1493,8 +1505,14 @@ def recurse_nodes(task, visited):
 
         form = DateTimeWithNumRunsForm(data={'base_date': max_date,
                                              'num_runs': num_runs})
-        external_logs = conf.get('elasticsearch', 'frontend')
+
         doc_md = wwwutils.wrapped_markdown(getattr(dag, 'doc_md', None), css_class='dag-doc')
+
+        task_log_reader = TaskLogReader()
+        handler = task_log_reader.log_handler
+        show_external_log_redirect = isinstance(handler, ExternalLoggingMixin)
```

Comment:
Can you create a property from this condition in TaskLogReader?

Reply:
Done.